app/event/raise-event.js

```diff
@@ -0,0 +1,24 @@
+const { PublishEvent } = require('ffc-pay-event-publisher')
+const { eventTopic } = require('../config/mq-config')
+
+const raiseEvent = async (event, status = 'success', error = '') => {
```

Comment:
`error` parameter isn't used so can be removed.

Reply:
removed